### ISSUE TITLE
SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable - false positive

### ISSUE THREAD

**Issue by bkdotcom** [slevomat/coding-standard#1733]
getting a "UnusedVariable" false positive 
likely doesn't recognize it's being assigned via the elvis / conditional / ternary operator

```php
        $numericIndex = 0;
        while ($this->registeredPlugins->valid()) {
            $plugin = $this->registeredPlugins->current();
            $pluginName = \array_search($plugin, $this->namedPlugins, true) ?: $numericIndex++;
            $this->registeredPlugins->next();
            $plugins[$pluginName] = $plugin;
        }
```

> Unused variable $numericIndex

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/1136fb939e14d2c1bfdf90fcc2099239dd16bb0e

**Comment by bkdotcom** [comment]
@kukulich  THANK YOU!
